package.json

```diff
@@ -19,12 +19,13 @@
     "compile:prod": "cross-env NODE_ENV=production parallel-webpack --config app/webpack.production.js",
     "build:dev": "yarn compile:dev && cross-env NODE_ENV=development ts-node script/build.ts",
     "build:prod": "yarn compile:prod && cross-env NODE_ENV=production ts-node script/build.ts",
-    "package": "ts-node script/package.ts",
+    "package": "ts-node -P script script/package.ts",
     "generate-octicons": "ts-node script/generate-octicons.ts",
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
-    "lint": "yarn compile:tslint && yarn tslint && yarn eslint-check && yarn eslint",
-    "tslint": "tslint ./{scripts,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
+    "compile:script": "tsc -p script",
```

Comment:
Just to make sure I understand, this is really to type check the scripts? `compile` makes me think there's actually gonna be output. What do you think about changing the name?

Reply:
Yeah, this is the tricky part.

We use `ts-node` when invoking our scripts so we don't need to emit the JS files to disk, and I don't see the value in changing that. But we also want to catch this before we have to invoke the script, so "compiling" felt like the closest term here (and we're leveraging `tsc` here so I'm somewhat keen to stick to that word).

If there's a more appropriate name here ("lint" doesn't feel right, and we already have a lot of linting going on) then I'm fine with renaming it.

Reply:
Eh, I'll refrain from bike shedding it.